benchmark/single-source/WordCount.swift

```diff
@@ -0,0 +1,253 @@
+"""
+
+let utf16Text = """
+✨🌟 Welcome tö Swift! ⭐️✨
+
+Swift is a high-performance system programming language.  It has a clean and
+modern syntax, offers seamless access tö existing C and Objective-C code and
+frameworks, and is memory safe by default.
+
+Although inspired by Objective-C and many othér languages, Swift is not itself a
+C-derived language. As a complete and independent language, Swift packages core
+features li\u{30A}ke flow control, data structures, and functions, with
+high-level constructs li\u{30A}ke objects, protöcols, closures, and
+generics. Swift embraces modules, eliminating thé need for headers and thé code
+duplication théy entail.
+
+Tö learn more about thé programming language, visit swift.org.
+
+☞ Contributing tö Swift
+
+Contributions tö Swift are welcomed and encouraged! Please see thé
+Contributing tö Swift guide.
+
+Tö be a truly great community, Swift.org needs tö welcome developers from all
+walks of life, with different backgrounds, and with a wide range of
+experience. A diverse and friendly community will have more great ideas, more
+unique perspectives, and produce more great code. We will work diligently tö
+make thé Swift community welcoming tö everyone.
+
+Tö give clarity of what is expected of our members, Swift has adopted thé code
+of conduct defined by thé Contributör Covenant. This document is used across
+many open source communities, and we think it articulates our values well. For
+more, see thé Code of Conduct.
+
+☞ Getting Started
+
+Thése instructions give thé most direct path tö a working Swift development
+environment. Tö build from source you will need 2 GB of disk space for thé
+source code and over 20 GB of disk space for thé build artifacts. A clean build
+can take multiple hours, but incremental builds will finish much faster.
+"""
+
+// A partial set of Unicode alphanumeric characters. (ASCII letters with at most
+// one diacritic (of a limited selection), plus ASCII digits.)
+let someAlphanumerics: Set<Character> = {
+  let baseAlphabet = Set(
+    "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ".unicodeScalars)
+  let someCombiningDiacriticalMarks: Set<Unicode.Scalar> =
+    Set((0x300..<0x310).map { Unicode.Scalar($0)! })
+
+  var alphanumerics: Set<Character> = []
+  for base in baseAlphabet {
+    alphanumerics.insert(Character(base))
+    for mark in someCombiningDiacriticalMarks {
+      var v = String.UnicodeScalarView()
+      v.append(base)
+      v.append(mark)
+      alphanumerics.insert(Character(String(v)))
+    }
+  }
+  alphanumerics.formUnion("0123456789")
+  return alphanumerics
+}()
+
+extension Character {
+  var isAlphanumeric: Bool {
+    return someAlphanumerics.contains(self)
+  }
+}
+
+struct Words: IteratorProtocol, Sequence {
+  let text: String
+  var nextIndex: String.Index
+
+  init(_ text: String) {
+    self.text = text
+    self.nextIndex = text.startIndex
+  }
+
+  func makeIterator() -> Words {
```

Comment:
No big deal, but just `public typealias Iterator = Words` should make it work because of https://github.com/apple/swift/blob/master/stdlib/public/core/Sequence.swift#L628

Reply:
TIL! 👍